web_ui/frontend/app/(login)/login/page.tsx

```diff
@@ -78,86 +76,111 @@ export default function Home() {
                 }
             }
 
-        } catch {
+        } catch (e) {
+            console.error(e)
             setLoading(false)
             setError("Could not connect to server")
         }
     }
 
     function onSubmit(e: React.FormEvent<HTMLFormElement>) {
         e.preventDefault()
-
         submit(password)
     }
 
+    const LoginComponent = (
+        <form onSubmit={onSubmit} action="#">
+            <Box display={"flex"} justifyContent={"center"}>
+                <PasswordInput
+                    FormControlProps={{
+                        sx: {width: "50%"},
+                    }}
+                    TextFieldProps={{
+                        InputProps: {
+                            sx: {width: "50%"},
+                            onChange: (e) => {
+                                setPassword(e.target.value)
+                                setError(undefined)
+                            }
+                        }
+                    }}
+                />
+            </Box>
+            <Box display={"flex"} flexDirection={"column"}>
+                <Grow in={error !== undefined}>
+                    <Typography
+                        textAlign={"center"}
+                        variant={"subtitle2"}
+                        color={"error.main"}
+                        mb={1}
+                    >
+                        {error}
+                    </Typography>
+                </Grow>
+                <LoadingButton
+                    variant="outlined"
+                    sx={{margin: "auto"}}
+                    color={"primary"}
+                    type={"submit"}
+                    loading={loading}
+                >
+                    <span>Confirm</span>
```

Comment:
```suggestion
                    <span>Login</span>
```

I think Brian wants this to be `Login` or `Next`